sycl/include/sycl/accessor.hpp

```diff
@@ -1861,28 +1861,28 @@ class __SYCL_EBO __SYCL_SPECIAL_CLASS __SYCL_TYPE(accessor) accessor :
 
   iterator begin() const noexcept {
     return iterator::getBegin(
-        get_pointer(),
+        getPointerAdjusted(),
```

Comment:
Why this change? Is that a bugfix?

Reply:
This change was done because `get_pointer()` had deprecated template specialization, which caused certain tests using iterators to fail after deprecated warnings were re-enabled.